rest/route/admin_test.go

```diff
@@ -307,7 +307,7 @@ func TestRestartVersionsRoute(t *testing.T) {
 	projectRef := &model.ProjectRef{
 		Id: "my-project",
 		CommitQueue: model.CommitQueueParams{
-			PatchType: commitqueue.PRPatchType,
+			PatchType: commitqueue.SourcePullRequest,
```

Comment:
After this change bakes in, will we remove PatchType from this struct?

Reply:
yep, the second part will be done in EVG-13750